portfolio/src/main/webapp/index.html

```diff
@@ -27,6 +27,9 @@ <h1>Gabi Newman's Portfolio</h1>
       <p>Click here to get a random fact about Gabi:</p>
       <button onclick="addRandomFact()">Fun Fact!</button>
       <div id="fact-container"></div>
+      <p>Click here to get a special message from the Java servlet:</p>
+      <button onclick="getHello()">Click me!</button>
```

Comment:
Can you rename getHello to something like getMessageFromServlet? Doing so lets you change the contents of the function if you ever want to return something more than a "hello" in the future

Reply:
That makes sense!